api/lib/resolvers.js

```diff
@@ -263,6 +244,23 @@ const resolvers = (networkList, notificationController) => ({
             validator.operatorAddress
           ].profile
         : undefined
+    },
+    feed: async (validator, _, { dataSources }) => {
+      const networkList = await db.getNetworks()
```

Comment:
there is no need to call the db here. we have the network list already in this file

Reply:
true. This is leftover from the other location